doc/classes/RigidBody.xml

```diff
@@ -324,19 +324,19 @@
 	</methods>
 	<members>
 		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp">
-			Dampens rotational forces of the Rigid body by the 'angular_damp' rate.
+			Dampens Rigid Body's rotational forces.
 		</member>
 		<member name="angular_velocity" type="Vector3" setter="set_angular_velocity" getter="get_angular_velocity">
-			The current rotational velocity of the Rigid body
+			Rigid Body's rotational velocity.
 		</member>
 		<member name="axis_lock" type="int" setter="set_axis_lock" getter="get_axis_lock" enum="RigidBody.AxisLock">
 			Locks the rotational forces to a particular axis, preventing rotations on other axes.
 		</member>
 		<member name="bounce" type="float" setter="set_bounce" getter="get_bounce">
-			Bounciness of the Rigid body.
+			Rigid Body's bounciness.
 		</member>
 		<member name="can_sleep" type="bool" setter="set_can_sleep" getter="is_able_to_sleep">
-			If true, the Rigid body will no longer calculate forces when there is no movement and will act as a static body. It will wake up when other forces are applied through other collisions or when the 'apply_impulse' method is used.
+			If true, the Rigid body will not calculate forces and will act as a static body while there is no movement. It will wake up when other forces are applied through other collisions or when the 'apply_impulse' method is used.
```

Comment:
Naming again. Either say "the body will..." or "the RigidBody will..."

Surround "true" and "apply_impulse" in code blocks: [code]true[/code]

There should be no comma after "If [code]true[/code]"

Add default value.